features/roomlist/impl/src/main/kotlin/io/element/android/features/roomlist/impl/components/RoomListTopBar.kt

```diff
@@ -107,7 +107,7 @@ private fun DefaultRoomListTopBar(
                     modifier = Modifier.testTag(TestTags.homeScreenSettings),
                     onClick = onOpenSettings
                 ) {
-                    Avatar(matrixUser.avatarData, contentDescription = stringResource(StringR.string.common_settings))
+                    Avatar(matrixUser.getAvatarData(), contentDescription = stringResource(StringR.string.common_settings))
```

Comment:
I wonder if we should be using `derivedStateOf` to avoid generating an `AvatarData` object in each recompilation. Maybe it's not worth the trouble.

Reply:
Yeah, fair point. Done.